src/main/java/org/jenkinsci/plugins/workflow/cps/CpsScript.java

```diff
@@ -91,6 +91,15 @@ public CpsScript() throws IOException {
     @Override
     public final Object invokeMethod(String name, Object args) {
         // TODO probably better to call super method and only proceed here incase of MissingMethodException:
+        
+        // check for user defined closures in the script binding
+        if (getBinding().hasVariable(name)){
+            if (getBinding().getVariable(name) instanceof CpsClosure2){
+		CpsClosure2 local_closure = (CpsClosure2) getBinding().getVariable(name);
```

Comment:
Nitpick: why not store `getBinding().getVariable(name)` to a local variable so we don't repeatedly fetch?